src/app/main.cpp

```diff
@@ -799,13 +813,7 @@ int main( int argc, char *argv[] )
   myApp.setWindowIcon( QIcon( QgsApplication::appIconPath() ) );
 #endif
 
-
-  //
-  // Set up the QSettings environment must be done after qapp is created
-  QCoreApplication::setOrganizationName( QgsApplication::QGIS_ORGANIZATION_NAME );
-  QCoreApplication::setOrganizationDomain( QgsApplication::QGIS_ORGANIZATION_DOMAIN );
-  QCoreApplication::setApplicationName( QgsApplication::QGIS_APPLICATION_NAME );
-  QCoreApplication::setAttribute( Qt::AA_DontShowIconsInMenus, false );
+  myApp.setWindowIcon( QIcon( QgsApplication::appIconPath() ) );
```

Comment:
What's happening here? looks like a duplicate of the code which is just run above.